example/bookstore/v1/bookstore.yaml

```diff
@@ -8,6 +8,18 @@ resources:
         type: STRING
         number: 1
         required: true
+      price:
+        type: FLOAT
+        number: 2
+        required: true
+      published:
+        type: BOOLEAN
+        number: 3
+        required: true
+      edition:
+        type: INT32
```

Comment:
I think it's probably better to leave it this way, but as a note there is no int32 type really, and the maximum size 2^53-1 so int64 isn't possible: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER.

I *think* we should probably support it anyway, and maybe implement an int64 as a string in javascript for safety's sake. @hudlow might have thoughts on this.

Reply:
Made #21 to discuss.